README.md

```diff
@@ -56,3 +56,9 @@ task | function
 --- | ---
 `npm run start` | Start local server with your optimized version
 `npm run ngrok` | Create a proxy to your local server with [ngrok](ngrok https://www.npmjs.com/package/ngrok).
+
+
+## Documented limitations
```

Comment:
Everything in a readme is documented, so I'd simplify this to `## Limitations` or `## Known limitations`.